package.xml

```diff
@@ -29,1058 +29,869 @@
-    <file baseinstalldir="/" name="src/php/ext/grpc/byte_buffer.h" role="src" />
-    <file baseinstalldir="/" name="src/php/ext/grpc/call.h" role="src" />
-    <file baseinstalldir="/" name="src/php/ext/grpc/call_credentials.h" role="src" />
-    <file baseinstalldir="/" name="src/php/ext/grpc/channel.h" role="src" />
-    <file baseinstalldir="/" name="src/php/ext/grpc/channel_credentials.h" role="src" />
-    <file baseinstalldir="/" name="src/php/ext/grpc/completion_queue.h" role="src" />
-    <file baseinstalldir="/" name="src/php/ext/grpc/php7_wrapper.h" role="src" />
-    <file baseinstalldir="/" name="src/php/ext/grpc/php_grpc.h" role="src" />
-    <file baseinstalldir="/" name="src/php/ext/grpc/server.h" role="src" />
-    <file baseinstalldir="/" name="src/php/ext/grpc/server_credentials.h" role="src" />
-    <file baseinstalldir="/" name="src/php/ext/grpc/timeval.h" role="src" />
-    <file baseinstalldir="/" name="src/php/ext/grpc/version.h" role="src" />
-    <file baseinstalldir="/" name="third_party/address_sorting/address_sorting_internal.h" role="src" />
-    <file baseinstalldir="/" name="third_party/address_sorting/include/address_sorting/address_sorting.h" role="src" />
-    <file baseinstalldir="/" name="third_party/address_sorting/address_sorting.c" role="src" />
-    <file baseinstalldir="/" name="third_party/address_sorting/address_sorting_posix.c" role="src" />
-    <file baseinstalldir="/" name="third_party/address_sorting/address_sorting_windows.c" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/alloc.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/atm.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/atm_gcc_atomic.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/atm_gcc_sync.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/atm_windows.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/cpu.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/log.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/log_windows.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/port_platform.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/string_util.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/sync.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/sync_custom.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/sync_generic.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/sync_posix.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/sync_windows.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/thd_id.h" role="src" />
-    <file baseinstalldir="/" name="include/grpc/support/time.h" role="src" />
+      
```

Comment:
There's somehow one extra newline here. It's an xml file so it probably doesn't matter. But it would be nice to figure out why. Every other file has equal number of lines of additions and subtractions except this.

Reply:
Thanks! I removed new new line by adding \ at the end of the line.

Reply:
Thanks! I managed to remove this new line by adding \ at the end of the line in the template file.